zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -0,0 +1,503 @@
+        self.capacity = 0
+        self.usedSize = 0
+
+
+class GetFactsRsp(AgentResponse):
+    def __init__(self):
+        super(GetFactsRsp, self).__init__()
+        self.version = None
+
+
+def replyerror(func):
+    @functools.wraps(func)
```

Ruff (0.8.2)

`functools` may be undefined, or defined from star imports

(F405)